plane/src/database/backend_key.rs

```diff
@@ -105,22 +135,23 @@ pub enum BackendKeyHealth {
 }
 
 pub struct BackendKeyResult {
-    pub id: BackendKeyId,
+    pub id: BackendName,
     pub tag: String,
-    backend_id: BackendName,
-    last_renewed: DateTime<Utc>,
+    pub token: i64,
+    pub key: String,
+    expires_at: DateTime<Utc>,
     as_of: DateTime<Utc>,
 }
 
 impl BackendKeyResult {
     pub fn key_health(&self) -> BackendKeyHealth {
-        let key_age = (self.as_of - self.last_renewed).num_seconds();
+        let key_age = (self.as_of - self.expires_at).num_seconds();
```

Comment:
Is this logic correct? Doesn't `as_of - expires_at` give you the amount of time since the key expired?

Reply:
fixed.